nodejs-backend/reload_params.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-source /opt/ros/noetic/setup.bash
-source /home/mirte/mirte_ws/devel/setup.bash
-rosnode kill /mirte_telemetrix_mirte
-rosparam delete /mirte
-rosparam load /home/mirte/mirte_ws/src/mirte-ros-packages/mirte_telemetrix/config/mirte_user_config.yaml mirte
-rosrun mirte_telemetrix ROS_telemetrix_api.py  __name:=mirte_telemetrix_mirte
-
+source /opt/ros/humble/setup.bash
+source /home/mirte/mirte_ws/install/setup.bash
+# rosnode kill /mirte_telemetrix_mirte
+# rosparam delete /mirte
+# rosparam load /home/mirte/mirte_ws/src/mirte-ros-packages/mirte_telemetrix/config/mirte_user_config.yaml mirte
+# rosrun mirte_telemetrix ROS_telemetrix_api.py  __name:=mirte_telemetrix_mirte
+sudo systemctl restart mirte-ros
```

Comment:
We need to check this later,  is the mirte-master ros service also called mirte-ros?